Source/Tools/FEXLoader/FEXLoader.cpp

```diff
@@ -404,10 +403,14 @@ int main(int argc, char** argv, char** const envp) {
   FEXCore::Profiler::Init();
   FEXCore::Telemetry::Initialize();
 
-  RootFSRedirect(&Program.ProgramPath, LDPath());
-  InterpreterHandler(&Program.ProgramPath, LDPath(), &Args);
+  // Program.ProgramPath is highly likely to be prefixed with FEX's rootfs. Get rid of that.
```

Comment:
> // Program.ProgramPath is highly likely to be prefixed with FEX's rootfs

Why?

Reply:
Do I need to explain again in the comment that execve will always have the full host qualified path to the executable inside the rootfs?